cluster-provision/k8s/1.32/k8s_provision.sh

```diff
@@ -64,7 +64,7 @@ function pull_container_retry() {
     fi
 }
 
-export CRIO_VERSION=1.31
+export CRIO_VERSION=1.32
```

Comment:
Can we somehow derive this value from the k8s version? Is there some env var that gives it away? maybe the path?